interface/wx/desktopenv.h

```diff
@@ -0,0 +1,28 @@
+//////////////////////////////////////////////////////////////////////////////
+// Name:        interface/wx/desktopenv.h
+// Purpose:     wxDesktopEnv class documentation
+// Author:      Igor Korot
+// Created:     2021-09-05
+// Copyright:   (c) 2021 wxWidgets development team
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+/**
+@class wxDesktopEnv
+
+wxDesktopEnv contains the function for moving the file/folder to Trash
+or Recycle Bin. The call to this function is synchronous, which means that
+application will wait for the move to finish.
```

Comment:
There is no real need to mention that a function is sync, they all are by default.